nipype/pipeline/engine/workflows.py

```diff
@@ -930,13 +911,13 @@ def _get_dot(self, prefix=None, hierarchy=None, colored=False,
             prefix = '  '
         if hierarchy is None:
             hierarchy = []
-        colorset = ['#FFFFC8',                       # Y
-                    '#0000FF', '#B4B4FF', '#E6E6FF', # B
-                    '#FF0000', '#FFB4B4', '#FFE6E6', # R
-                    '#00A300', '#B4FFB4', '#E6FFE6', # G
-                    '#0000FF', '#B4B4FF'] # loop B
+        colorset = ['#FFFFC8',                        # Y
+                    '#0000FF', '#B4B4FF', '#E6E6FF',  # B
+                    '#FF0000', '#FFB4B4', '#FFE6E6',  # R
+                    '#00A300', '#B4FFB4', '#E6FFE6',  # G
+                    '#0000FF', '#B4B4FF']             # loop B
         if level > len(colorset) - 2:
-            level = 3 # Loop back to blue
+            level = 3  # Loop back to blue
```

Comment:
extra space?

Reply:
Nope, apparently there is some PEP somewhere advising to have 2 spaces before the inline comments